src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -814,9 +826,11 @@ StDebugger >> initializeWindow: aWindowPresenter [
 	super initializeWindow: aWindowPresenter.
 	aWindowPresenter
 		title: self debuggerActionModel statusStringForContext;
-		initialExtent: self initialExtent;
+		initialExtent: self debuggerSettings lastKnownExtent;
+		initialPosition: self debuggerSettings lastKnownPosition;
 		whenClosedDo: [ self clear ].
-
+		
+	self initializeSettings: aWindowPresenter.
```

Comment:
I would inline the window extent setting here (by calling the settings)

Reply:
Done